lib/spacesuit/auth_middleware.ex

```diff
@@ -29,6 +31,24 @@ defmodule Spacesuit.AuthMiddleware do
     end
   end
 
+  # Quick access function for the application settings for this middleware
+  def session_service do
+    Application.get_env(:spacesuit, :session_service)
```

Comment:
Unless we plan to update this in place, memoizing it into an `@` module var seems like a good idea.